root/server/components.js

```diff
@@ -0,0 +1,198 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+module.exports = {
+  /*
+   * Any server-rendered page referenced via component_path in the
+   * stash must be listed here.
+   */
+  'account/applications/Edit': require('../account/applications/Edit'),
+  'account/applications/Index': require('../account/applications/Index'),
+  'account/applications/Register': require('../account/applications/Register'),
+  'account/applications/Remove': require('../account/applications/Remove'),
+  'account/applications/RevokeAccess': require('../account/applications/RevokeAccess'),
+  'account/Donation': require('../account/Donation'),
+  'account/EmailVerificationStatus': require('../account/EmailVerificationStatus'),
+  'account/LostPassword': require('../account/LostPassword'),
+  'account/LostPasswordSent': require('../account/LostPasswordSent'),
+  'account/LostUsername': require('../account/LostUsername'),
+  'account/LostUsernameSent': require('../account/LostUsernameSent'),
+  'account/Preferences': require('../account/Preferences'),
+  'account/PreferencesSaved': require('../account/PreferencesSaved'),
+  'account/ResetPasswordStatus': require('../account/ResetPasswordStatus'),
+  'account/sso/DiscourseRegistered': require('../account/sso/DiscourseRegistered'),
+  'account/sso/DiscourseUnconfirmedEmailAddress': require('../account/sso/DiscourseUnconfirmedEmailAddress'),
+  'area/AreaLabels': require('../area/AreaLabels'),
+  'area/NotFound': require('../area/NotFound'),
+  'artist/ArtistEvents': require('../artist/ArtistEvents'),
+  'artist/CannotSplit': require('../artist/CannotSplit'),
+  'artist/Merge': require('../artist/Merge'),
+  'artist/NotFound': require('../artist/NotFound'),
+  'artist/SpecialPurpose': require('../artist/SpecialPurpose'),
+  'cdtoc/NotFound': require('../cdtoc/NotFound'),
+  'collection/NotFound': require('../collection/NotFound'),
+  'edit/NotFound': require('../edit/NotFound'),
+  'entity/Aliases': require('../entity/Aliases'),
+  'entity/Details': require('../entity/Details'),
+  'entity/Subscribers': require('../entity/Subscribers'),
+  'entity/Tags': require('../entity/Tags'),
+  'elections/Index': require('../elections/Index'),
+  'elections/Nominate': require('../elections/Nominate'),
+  'elections/NotFound': require('../elections/NotFound'),
+  'elections/Show': require('../elections/Show'),
+  'event/NotFound': require('../event/NotFound'),
+  'genre/List': require('../genre/List'),
+  'instrument/List': require('../instrument/List'),
+  'instrument/List': require('../instrument/List'),
```

Comment:
There are some duplicates in these exports.

Reply:
Whoops, fixed